creations/espressif.md

```diff
@@ -21,3 +21,7 @@ Community Allocated Creation IDs for Espressif boards
 
 ## `0x0142_xxxx` - H2 dev boards
 *  `0x0142_0001` ESP32-H2-DevKitM-1
+
+## `0x00D0_xxxx` - WROVER-E (ESP32-D0WDQ6) dev boards
+*  `0x00D0-0001` ESP32-WROVER-DEV-CAM
```

Comment:
I don't think this should be under Espressif. Instead, make a new creation id for Freenove and make a creations/freenove.md for it.

(Espressif dev boards are documented here: https://www.espressif.com/en/products/devkits )